client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/ExternalServiceInfoUtils.java

```diff
@@ -30,7 +30,7 @@ public static String[] collectExternalServiceInfo(ExternalServiceInfo remoteHost
      * skip method replacement for some hostname, eg,
      */
     public static boolean skipHostnameOrIp(String hostname){
-        return hostname.isEmpty() || hostname.startsWith("localhost") || hostname.startsWith("0.0.0") || hostname.startsWith("127.0.0.1");
+        return hostname.isEmpty() || hostname.startsWith("localhost") || hostname.startsWith("0.0.0") || hostname.startsWith("127.0.0.1") || hostname.startsWith("docker.socket");
```

Comment:
not related to this PR, but should the `"127.0.0.1"` be replaced with `"127."?  or otherwise have an `equal` there?